scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/RawChange.java

```diff
@@ -67,6 +69,12 @@ default boolean isNull(String columnName) {
     
     boolean isNull(ChangeSchema.ColumnDefinition c);
     
+    default ByteBuffer getUnsafeBytes(String columnName) {

```

Comment:
Actually, when I started developing the library, I added `getUnsafeBytes` method.

However, @haaawk was against this (we discussed it several times) and I deleted it (it required writing those translation classes from library representation to driver3 representation):
https://github.com/scylladb/scylla-cdc-java/commit/862fa051b42d522ba113b2861d1188cc5aad0fad

Reply:
My thought is that the user should be able to pick his transport for the CDC (at least v3 or v4 cql), and utilize non-marshalled copying of data via above. Another "reason" is to make it imminent that values are lazy, not force-marshalled. To make wrapper objects more light weight.

Reply:
@haaawk What's your opinion?